package/samplers/hill_climb_search/hill_climb_search.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+from typing import Any
+
+import numpy as np
+import optuna
+import optunahub
+
+
+class HillClimbSearch(optunahub.samplers.SimpleBaseSampler):
```

Comment:
Could you add the `Sampler` suffix to the class name to follow the naming convention? I think this would help users recognize it as an Optuna sampler.

Alternatively, we could assign a slightly shorter name, like `HillClimbingSampler`.

```suggestion
class HillClimbSearchSampler(optunahub.samplers.SimpleBaseSampler):
```